source/config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+
+  resources :urls
```

Comment:
Best practice is to only create the routes you need. `resources :urls, except: [:destroy]` in this case.